tests/testthat/test-other.R

```diff
@@ -157,3 +157,44 @@ test_that("enforceMinCellValue works with vector of minimum values", {
 
   expect_equal(result$a, c(1, 2, 3, 4, 5))
 })
+
+test_that("Creating and checking externalConceptCounts table", {
+  if (dbms == "sqlite") {
+    # Creating externalConceptCounts
+    sql_lite_path <- file.path(test_path(), databaseFile)
+    connectionDetails <- createConnectionDetails(dbms= "sqlite", server = sql_lite_path)
```

Comment:
We may want to build tests out for real databases but this can likely wait - I think adding a pre-computed counts table should speed up the other unit tests